bigtable/google/cloud/bigtable/table.py

```diff
@@ -199,6 +207,60 @@ def row(self, row_key, filter_=None, append=False):
         else:
             return DirectRow(row_key, self)
 
+    def append_row(self, row_key):
+        """Create a :class:`~google.cloud.bigtable.row.AppendRow` associated with this table.
```

Comment:
We should make sure these comments reflect why you would choose one of the rows over another.

Reply:
Not sure, I'm getting. You want something like: you read method's docs, and you don't need to read row class'es docs, 'cause method's docs will tell you everything you need? So classes should be described a little in method's docs?